core/domain/feedback_jobs_one_off.py

```diff
@@ -100,3 +103,64 @@ def map(item):
     @staticmethod
     def reduce(key, value):
         pass
+
+
+class SuggestionMigrationOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Should this be a part of a `suggestion_jobs_one_off.py`? (/cc @seanlip)

Reply:
Ah I'm not entirely sure for this comment. Will go with your opinion. I put in in this file as the model it is mapping over is a feedback model.

Reply:
Sorry, I missed this comment (in general, please don't hesitate to ping me if you don't get a response within 24 hours).

I don't have a preference either way.